isis/src/control/apps/findfeatures/GenericTiledTransform.cpp

```diff
@@ -0,0 +1,175 @@
+/** This is free and unencumbered software released into the public domain.
```

Comment:
A lot of this code was ripped directly from https://gist.github.com/kinchungwong/141bfa2d996cb5ae8f42.  Should there be a "credit" comment?

Reply:
Add a link to the gist on the class docs for GenericTilesTransform.

Reply:
I think there is a copyright in the headers for those files, likely a problem for use in ISIS?

Reply:
Yes, the copyright on the code is a problem. We can't just use the code and will have to write this from scratch :(

Reply:
The Gist indicates that this was to get around a bug 8 years ago.  Has the bug not been fixed that we could benefit without having to rewrite?

Is it that an approach or technique in that code was mimicked?  That's different from cutting and pasting (which, 
sadly the "ripped directly" implies), so maybe you do need to clean-room reimplement this.  Bummer.

Reply:
Unfortunately the bug was not fixed as it has to do with some fundamental choices that were made in OpenCV's remap function.

This PR completely copies a handful of functions from the gist, which I think would violate the copyright. It doesn't take all of the code, but a good chunk of it.